.github/workflows/publish-bulk-cdk.yml

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+name: Publish Bulk CDK
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "airbyte-cdk/bulk"
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: "Repo to check out code from. Defaults to the main airbyte repo."
+        type: choice
+        required: true
+        default: airbytehq/airbyte
+        options:
+          - airbytehq/airbyte
+      build-number:
+        description: "Build Number"
+        required: false
+        type: number
+
+concurrency:
+  group: publish-bulk-cdk
```

Comment:
You could add the branch name to the group so that a manual CDK publish from a branch for testing would not wait on a publish from master (e vice versa).

Reply:
I don't think we'll ever publish from a branch unless it's to recover from some weird mess-up. I may be wrong, we'll see.